<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usage_plan</class>
 <widget class="QWidget" name="usage_plan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,3,15">
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="time">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>09：30</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>647</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="back">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 6px 12px;</string>
        </property>
        <property name="text">
         <string>返回首页</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="date_select" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="date0">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: none;
    border: none;
    color: black;
    padding: 8px 16px;

}

QPushButton:pressed {
    background-color: white;
 	color: blue;
}

QPushButton:checked {
    background-color: white;
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>12月15日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date1">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: black;
    padding: 8px 16px;
}

QPushButton:pressed {
    background-color: white;
 	color: blue;
}

QPushButton:checked {
    background-color: white;
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>12月15日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: black;
    padding: 8px 16px;
}
QPushButton:pressed {
    background-color: white;
 	color: blue;
}

QPushButton:checked {
    background-color: white;
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>12月15日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: black;
    padding: 8px 16px;
}

QPushButton:pressed {
    background-color: white;
 	color: blue;
}

QPushButton:checked {
    background-color: white;
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>10月23日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: black;
    padding: 8px 16px;
}

QPushButton:pressed {
    background-color: white;
 	color: blue;
}

QPushButton:checked {
    background-color: white;
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>10月24日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date5">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: black;
    padding: 8px 16px;
}

QPushButton:pressed {
    background-color: white;
 	color: blue;
}

QPushButton:checked {
    background-color: white;
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>10月25日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date6">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: black;
    padding: 8px 16px;
}

QPushButton:pressed {
    background-color: white;
 	color: blue;
}

QPushButton:checked {
    background-color: white;
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>10月26日</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea#scrollArea{
	background: transparent;
	border: none;
}</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>385</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scrollAreaWidgetContents
{
	background: transparent;
}</string>
      </property>
      <layout class="QGridLayout" name="card_layout" rowstretch="0" rowminimumheight="0" columnminimumwidth="0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>100</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>100</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <property name="horizontalSpacing">
        <number>60</number>
       </property>
       <property name="verticalSpacing">
        <number>40</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
